site/content/entry/yuzu-progress-report-jun-2023/index.md

```diff
@@ -0,0 +1,735 @@
+Newcomer [mrcmunir](https://github.com/mrcmunir) decided to change this and {{< gh-hovercard "10635" "mark the extension as optional," >}} letting the most optimal ARM boards for Switch emulation be able to enjoy some good yuzu gameplay.
+
+To close the graphics section, the best for last.
+toastUnlimited removed the use of a Windows only external memory Vulkan extension.
+Why? To make yuzu {{< gh-hovercard "10829" "compatible with Wine" >}} while using Vulkan!
+Why would someone run yuzu on Wine when a native and faster Linux version is available? We don’t know! But you can definitely do it now, given you build Wine from source yourself with support for Vulkan child windows.
+We only intend to use this for debugging MinGW releases—for end users, we won't provide any support for running releases in Wine.
+
+This is a fun case where all developers asked your writer to skip mentioning this change because it was considered redundant for this report.
+To them I say: YOLO.
+
+## Android additions
+
+The new supported platform in town has a lot of improvements to boast of, but the most important one gets the honour of starting the section.
+
+Thanks to the joint work of byte[] and [GPUCode](https://github.com/GPUCode), and the help from [bylaws](https://github.com/bylaws) who allowed us to use Skyline’s BCn texture decoder, yuzu can now properly advertise {{< gh-hovercard "10837" "support for Mali GPUs!" >}}
+
+[Kept you waiting, huh.](https://www.youtube.com/watch?v=6EBuKP-uc94)
+
+This means users of Mediatek, old Samsung Exynos, and Google Tensor CPUs [(among others)](https://en.wikipedia.org/wiki/Mali_(processor)#Implementations) can run their games on yuzu as long as they have a [G series Mali GPU](https://en.wikipedia.org/wiki/Mali_(processor)#Variants), for example the Mali-G710 on the Pixel 7 series.
+
+{{< single-title-imgs
+    "SUPER MARIO ODYSSEY and Pokémon Mystery Dungeon Rescue Team DX. Pics taken with a Pixel 7 Pro"
+    "./mali1.png"
+    "./mali2.png"
+    >}}
+
+This theoretically doubles the compatibility of yuzu’s Android builds, as Mali is the most common GPU family in the Android ecosystem.
+
+{{< single-title-imgs
+    "The Legend of Zelda: Skyward Sword HD, Advance Wars 1+2: Re-Boot Camp, and OCTOPATH TRAVELER. Pics taken with a Pixel 7 Pro"
+    "./mali3.png"
+    "./mali4.png"
+    "./mali5.png"
+    >}}
+
+Further testing with Mali revealed that while the hardware briefly had support for the VK_EXT_extended_dynamic_state2 extension in older Mali drivers, the implementation is actually completely broken.
+This is most likely the reason the extension was removed from current drivers. 
+To avoid users of affected drivers from suffering random crashes, byte[] {{< gh-hovercard "10790" "disabled the use of the extension" >}} on Mali hardware.
```

Comment:
```suggestion
To avoid users of affected drivers from getting no rendering output at all, byte[] {{< gh-hovercard "10790" "disabled the use of the extension" >}} on Mali hardware.
```